app/actions/ConfigActionCreators.js

```diff
@@ -0,0 +1,25 @@
+import constants from '../constants';
+import ConfigAPI from '../api/ConfigAPI';
+
+export default class ConfigActionCreators {
+  static getAppData() {
+    return (dispatch) => {
+      dispatch({ type: constants.FETCH_APP_DATA });
+    }
+  }
+  static fetchConfigData() {
```

Comment:
Thanks very much Sir. I noticed you didn't use the curly braces for arrow functions containing multiple lines of code, as a result eradicating the need for the `return` statement. Is this a preferred approach?